### DISCUSSION TITLE
Zed auto completion based on existing words in a file

### DISCUSSION THREAD

**Discussion by venkatb-zelar** [zed-industries/zed#17013] [Feature Requests]
Isn't zed supposed to show the existing words in a file as we type the same word again?

**Comment by bajrangCoder** [comment]
No, it isn't available.

Btw try simple-autocompletion extension which maybe helpful

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
